code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -47,6 +47,8 @@
 
 	var/datum/physiology/physiology
 
+	var/can_use_vault = TRUE
```

Comment:
just have a static list on the DNA vault of all users instead, and only allow people to use it if they're not in said list

Reply:
Didn't we fase out static vars for GLOBs :)

Reply:
har har har har har har very funny